addon/components/sl-calendar.js

```diff
@@ -523,17 +526,18 @@ export default Ember.Component.extend({
             }
 
             for ( let week = 0; week < 6; week++ ) {
-                let days = Ember.A();
+                const days = new Ember.A();
 
                 for ( let wday = 0; wday < 7; wday++ ) {
-                    let active = !inPreviousMonth && !inNextMonth &&
+                    const active = !inPreviousMonth && !inNextMonth &&
                         contentDates.hasOwnProperty( year ) &&
                         contentDates[ year ].hasOwnProperty( month ) &&
                         contentDates[ year ][ month ].hasOwnProperty( day );
 
                     days.push({
                         active,
-                        content: active ?  contentDates[ year ][ month ][ day ] : null,
+                        content: active ?
+                            contentDates[ year ][ month ][ day ] : null,
```

Comment:
Is the line breaking here because of line length or an ESLint rule?  Would think the `null` would be on its own line as well so as to not get visually lost